packages/aws-cdk-lib/aws-cloudfront/test/distribution.test.ts

```diff
@@ -1294,7 +1294,16 @@ describe('Distribution metrics tests', () => {
     })),
   ];
 
-  test.each(metrics)('get %s metric', (metric) => {
+  const defaultMetrics = [
+    { name: 'Requests', method: 'metricRequests', additionalMetricsRequired: false, errorMetricName: '' },
+    { name: 'BytesDownloaded', method: 'metricBytesDownloaded', additionalMetricsRequired: false, errorMetricName: '' },
+    { name: 'BytesUploaded', method: 'metricBytesUploaded', additionalMetricsRequired: false, errorMetricName: '' },
+    { name: 'TotalErrorRate', method: 'metricTotalErrorRate', additionalMetricsRequired: false, errorMetricName: '' },
+    { name: '4xxErrorRate', method: 'metric4xxErrorRate', additionalMetricsRequired: false, errorMetricName: '' },
+    { name: '5xxErrorRate', method: 'metric5xxErrorRate', additionalMetricsRequired: false, errorMetricName: '' },
+  ];
+
+  test.each(additionalMetrics.concat(defaultMetrics))('get %s metric', (metric) => {
```

Comment:
Based on comments you added, same of the metrics' default statistic are `@default - sum over 5 minutes`, but the test is expecting the metricObj to `statistic: 'Average'`.

Reply:
I made an error in the comment description. I have corrected it to 'default - average over 5 minutes' for all.

Reply:
According to the `CloudWatchConsoleServiceDirectory.json` file from [this repo](https://github.com/cdklabs/awscdk-service-spec/blob/main/sources/CloudWatchConsoleServiceDirectory/CloudWatchConsoleServiceDirectory.json), it seems that the previous comment were correct. 
```
"metrics": [
          {
            "id": "AWS::CloudFront::Distribution:Requests",
            "name": "Requests",
            "defaultStat": "Sum"
          },
          {
            "id": "AWS::CloudFront::Distribution:TotalErrorRate",
            "name": "TotalErrorRate",
            "defaultStat": "Average"
          },
          {
            "id": "AWS::CloudFront::Distribution:BytesDownloaded",
            "name": "BytesDownloaded",
            "defaultStat": "Sum"
          },
          {
            "id": "AWS::CloudFront::Distribution:BytesUploaded",
            "name": "BytesUploaded",
            "defaultStat": "Sum"
          },
          {
            "id": "AWS::CloudFront::Distribution:4xxErrorRate",
            "name": "4xxErrorRate",
            "defaultStat": "Average"
          },
          {
            "id": "AWS::CloudFront::Distribution:5xxErrorRate",
            "name": "5xxErrorRate",
            "defaultStat": "Average"
          }
        ]
}
```

Reply:
I think the correct thing to do here is to set the default statistic like [this](https://github.com/aws/aws-cdk/blob/0129288695c244b1c3ba8523f459669ffb7cd2c1/packages/aws-cdk-lib/aws-lambda/lib/function.ts#L700) and update the tests accordingly.

Reply:
Thank you for your suggestion. I agree with what you have said. 
I have modified some of the metrics default to 'sum'!